### ISSUE TITLE
Two gateways with phoscon/deConz - How to "ui" into the second?

### ISSUE THREAD

**Issue by bubffm** [ebaauw/homebridge-deconz#176]
Following situation after migrating from homebridge-hue to homebridge-deconz:

I have two gateways (raspberry's) with raspbee/deconz running. Only one of them has Homebridge / homebridge-deconz installed (same way as it was previously with homebridge-hue). Devices from both gateways are being properly exposed in homekit.

But:
I can perfectly "-ui" into the gateway with the homebridge installed to change loglevels etc. but I dont seem to be able to access the other gateway (the one without homebridge).

Any way t oget around this? I dont really want to install homebridge on the second device.

Many thanks

**Comment by ebaauw** [comment]
`ui get /gateways` lists the gateways.  Then use `ui get /gateways/`_id_`/devices` to list the devices for a given gateway.

**Comment by bubffm** [comment]
Many thanks. That gave me the accessories of the second gateway. I cn update them. 

Now trying to put LogLevel 0 against the second gateway.  I can update devices themselfs
Based on your example in dynamic config, I use `ui-D put /gateways/ID/ '{"logLevel": 0}`

However this does not return nything.

**Comment by ebaauw** [comment]
The `/settings` is only needed when calling the API directly through `curl` or Postman.   It’s just `ui -D put /gateways/`_id_` ’{"logLlevel": 0}’`

**Comment by bubffm** [comment]
I tried that as well. this gets me stuck on that little "arrow" out of which I can only exit via CTRL+C
'ui -D put /gateways/id ’{"logLevel": 0}’.  Response: '>'
<img width="597" alt="Bildschirmfoto 2023-10-09 um 11 52 55" src="https://github.com/ebaauw/homebridge-deconz/assets/49747999/f1dbefa6-279d-4778-a5a1-43110265f9b7">

**Comment by ebaauw** [comment]
Your s8ngle quotes don’t match.  Make sure they’re not beautified.

**Comment by bubffm** [comment]
Oh, dear...   Corrected that one, but different Error message now.  Typed manually into terminal to avoid further beautifications.:

<img width="638" alt="Bildschirmfoto 2023-10-09 um 12 09 07" src="https://github.com/ebaauw/homebridge-deconz/assets/49747999/8baa0741-6b83-427c-a458-0ac158ec8e23">

**Comment by ebaauw** [comment]
You put a space in between the resource, causing `ui` to see the second part as the (invalid json) body.

**Comment by bubffm** [comment]
Here we go!  Thank you very much!

**Comment by bubffm** [comment]
Again issues with ui, I am afraid.  After I test that everything is working, I moced from my old raspi3/raspBee1 to a new raspi4/raspBee2 setup.  All is working fine.  Zigbee devices all showing up. But I want to stope that the new gateway (raspbee2) throws so many log entries.   Did a ping against the gateway and received a "pong" back.

But trying to set loglevel via "ui" gets me this error. I checked for beautifications, but all looks fine to me:
<img width="669" alt="Bildschirmfoto 2023-10-14 um 16 57 10" src="https://github.com/ebaauw/homebridge-deconz/assets/49747999/9e2b64a2-259a-4b04-8c05-e6f5ed5969ee">

Any idea whats wrong? Could there be a rights issue?
